turbinia/e2e/googleclouddisk.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# This Turbinia e2e test will test the googleclouddisk functionality.
+# It will try to process a disk created from the image 'test-disk2'.
+
+
+TURBINIA_CLI="turbiniactl"
+MAIN_LOG="main.log"
+STATS_LOG="stats.log"
+DETAIL_LOG="reqdetails.log"
+OUT_TGZ="reqresults.tgz"
+DISK="test-disk2"
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide project and zone."
```

Comment:
Very minor nit: `please provide the project and zone name.`

Reply:
See reply above.